mappings/net/minecraft/client/tutorial/BundleTutorial.mapping

```diff
@@ -0,0 +1,13 @@
+CLASS net/minecraft/class_5829 net/minecraft/client/tutorial/BundleTutorial
+	FIELD field_28798 manager Lnet/minecraft/class_1156;
+	FIELD field_28799 options Lnet/minecraft/class_315;
+	FIELD field_28800 toast Lnet/minecraft/class_372;
+	METHOD <init> (Lnet/minecraft/class_1156;Lnet/minecraft/class_315;)V
+		ARG 1 manager
+		ARG 2 options
+	METHOD method_33701 start ()V
+	METHOD method_33702 onRegularSlotClick (Lnet/minecraft/class_1799;Lnet/minecraft/class_1799;Lnet/minecraft/class_5536;)V
```

Comment:
It's a bit unclear how a "regular" slot click is different from just a slot click. Unqualified would be expected to be regular, so it is potentially confusing.

Reply:
I find no good term to summarize those non-quick craft non shift click etc clicks. It is just an adhoc created by mojang so they can call the bundle tutorial

Reply:
Think it is actually pickup slot action type click. rename coming